app/forms/work_packages/types/subject_configuration_form.rb

```diff
@@ -51,12 +51,13 @@ class SubjectConfigurationForm < ApplicationForm
         end
 
         subject_form.group(data: { "admin--subject-configuration-target": "patternInput" }) do |toggleable_group|
-          toggleable_group.text_field(
+          toggleable_group.pattern_input(
             name: :pattern,
+            value: model.pattern,
```

Comment:
❓ is that needed? usually the form mapping should work, if the name is equal to the attribute of the model.